assets/airports/eidw.json

```diff
@@ -1,1329 +1,5168 @@
+    "GANET": ["N53d26m06.50", "W6d31m33.80"],
+    "GERVO": ["N53d05m29.00", "W6d30m24.00"],
+    "GIMRO": ["N53d39m10.00", "W5d44m55.00"],
+    "GINIS": ["N53.46080028", "W4.85797119"],
+    "GIRAS": ["N53d38m21.00", "W5d57m33.20"],
+    "GOTNA": ["N54.59476978", "W5.59808350"],
+    "GURGA": ["N53d06m55.00", "W6d50m00.00"],
+    "IDEXA": ["N53.47050160", "W5.16412354"],
+    "INKUR": ["N53d35m51.30", "W7d23m28.60"],
+    "INTOP": ["N53d13m27.40", "W6d06m32.40"],
+    "IOM"  : ["N54.06691032", "W4.76348877"],
+    "IRDEX": ["N53d11m45.10", "W6d03m50.10"],
+    "IRKUM": ["N52d59m48.00", "W5d22m39.00"],
+    "KANUS": ["N53d06m30.50", "W6d26m52.10"],
+    "KEPOR": ["N53d10m16.50", "W6d22m00.70"],
+    "KERAV": ["N53d37m42.70", "W5d45m57.30"],
+    "KISHA": ["N53d29m02.00", "W5d38m22.00"],
+    "KOGAX": ["N53d34m18.60", "W5d38m14.10"],
+    "KORAK": ["N53.39501275", "W7.79296875"],
+    "KUDOM": ["N53d29m25.80", "W5d33m14.30"],
+    "LAPMO": ["N53d24m11.00", "W5d56m44.10"],
+    "LEMTA": ["N53d43m34.30", "W6d03m11.60"],
+    "LIFFY": ["N53d28m48.30", "W5d30m00.00"],
+    "LIPGO": ["N53d03m50.10", "W5d30m00.00"],
+    "LIRGU": ["N54.50620027", "W6.43560789"],
+    "LOMVO": ["N53.27112647", "W5.17950440"],
+    "LUTIP": ["N53.25612201", "W4.73785401"],
+    "MAXEV": ["N53d24m33.50", "W6d03m17.30"],
+    "MORAG": ["N52d45m10.00", "W5d30m00.00"],
+    "NARMU": ["N53d26m43.20", "W5d51m34.30"],
+    "NEKIL": ["N53d31m16.60", "W6d40m23.60"],
+    "NELBO": ["N54.49003739", "W5.30273438"],
+    "NEPOD": ["N52d56m57.20", "W6d10m29.80"],
+    "NEVRI": ["N54d04m06.00", "W6d16m11.40"],
+    "NEXAT": ["N51.93884673", "W6.57550049"],
+    "NIMAT": ["N53d57m54.10", "W5d44m31.70"],
+    "NITIL": ["N53d28m00.00", "W6d37m55.00"],
+    "NUMPI": ["N54.14441939", "W5.59332276"],
+    "OE"   : ["N53.43010119", "W6.42773436"],
+    "OGTUG": ["N53.67002933", "W5.14727783"],
+    "OLAPO": ["N53d46m49.00", "W7d17m41.00"],
+    "OLONO": ["N52d43m23.10", "W6d46m44.40"],
+    "OP"   : ["N53.41378839", "W6.13833617"],
+    "ORVEN": ["N53d39m53.50", "W6d11m29.80"],
+    "OSGAR": ["N53d02m57.90", "W7d16m12.80"],
+    "OSLEX": ["N53d21m55.80", "W6d41m44.50"],
+    "OTNER": ["N53.28750762", "W6.30752562"],
+    "PEKOK": ["N53d07m39.30", "W5d34m00.80"],
+    "PELIG": ["N53.19967128", "W7.33337403"],
+    "PEMOB": ["N52.14393950", "W5.16137695"],
+    "PESIT": ["N52d23m56.60", "W5d45m24.00"],
+    "RAMOX": ["N53.66026517", "W4.83947754"],
+    "RIKUL": ["N53.05772344", "W8.34576416"],
+    "RINUS": ["N53.64420365", "W7.66223145"],
+    "RISAP": ["N53d26m27.40", "W6d38m13.20"],
+    "RONON": ["N53d42m33.90", "W6d36m19.20"],
+    "ROTEV": ["N54d01m43.70", "W6d03m58.40"],
+    "RUBAR": ["N53d08m08.90", "W5d48m05.40"],
+    "SHA  ": ["N52.72098925", "W8.88519287"],
+    "SIVNA": ["N53d11m52.30", "W5d38m27.70"],
+    "SOPEP": ["N53d21m05.90", "W5d52m29.40"],
+    "SORIN": ["N53d08m29.30", "W5d48m22.50"],
+    "STU  ": ["N51.99468997", "W5.04016113"],
```

Comment:
Here and with `SHA` are the reasons for the errors. The spaces were accidentally inserted on the wrong side.